epr.py

```diff
@@ -38,9 +38,30 @@
 locale.setlocale(locale.LC_ALL, "")
 # code = locale.getpreferredencoding()
 
-configfile = os.path.join(os.getenv("HOME"), ".epr")
-config = configparser.ConfigParser(strict=False)
-config.read(configfile)
+def config_path():
+    home = os.getenv("HOME")
+
+    # check for xdg support
+    xdg_vars = { k:v for k,v in os.environ.items() if k.startswith('XDG_') }
+    if not xdg_vars:
+        return os.path.join(os.getenv("HOME"), ".epr")
+
+    xdg_home = xdg_vars.get('XDG_CONFIG_HOME', os.path.join(home, ".config"))
```

Comment:
Why looping over all variables instead of just getting `XDG_CONFIG_HOME` value ?